ui/page/start_page.go

```diff
@@ -102,6 +120,41 @@ func (sp *startPage) OnNavigatedTo() {
 	}
 }
 
+func (sp *startPage) initSetupItems() {
+	radius := cryptomaterial.CornerRadius{
+		TopRight:    8,
+		TopLeft:     8,
+		BottomRight: 8,
+		BottomLeft:  8,
+	}
+
+	setupActions := []*setupAction{
+		{
+			title:     values.String(values.StrRecommended),
+			subTitle:  values.String(values.StrRecommendedContent),
+			clickable: sp.Theme.NewClickable(false),
+			border: cryptomaterial.Border{
+				Radius: radius,
+				Color:  sp.Theme.Color.DefaultThemeColors().White,
+				Width:  values.MarginPadding2,
+			},
+			width: values.MarginPadding110,
```

Comment:
Since this is thesame in both setup action options, I really don't think they are needed. What you need to do is define a common layout with the border parameter and pass the relevant setupAction parameters to it. 

```
setupActionLayout(gtx, item setupAction, func(gtx C) D  {
    // the setup layout border codes go in here
})
```

Reply:
This is not yet resolved.

Reply:
Done.